app.js

```diff
@@ -37,6 +38,20 @@ app.use(express.urlencoded({ extended: false }))
 app.use(cookieParser(process.env.app_session_secret))
 app.use(require('./config/i18n.config').init)
 
+// CSRF setup
+app.use(
+  csrf({
+    cookie: true,
+    signed: true,
+  }),
+)
+
+// append csrfToken to all responses
+app.use(function(req, res, next) {
+  res.locals.csrfToken = req.csrfToken()
```

Comment:
This is done so that the templates have access to the token, right?

Reply:
Correct